src/components/shared/FilterNavigatorBar.css

```diff
@@ -38,6 +38,14 @@
 
 .filterNavigatorBarItemContent {
   overflow: hidden;
+
+  /* The 8px are used by the box-shadow when the button receives focus */
```

Comment:
can you add a comment that these lines are mostly to override the default browser styles for `<button>`?